### ISSUE TITLE
Error: Can't resolve 'papaparse/papaparse.min.js' wiht ngx-papaparse@6

### ISSUE THREAD

**Issue by mgoku** [alberthaff/ngx-papaparse#102]
**Describe the bug**

Today I upgraded ngx-papaparse to version 6.0 and then my build failed with error message: 

```
./node_modules/ngx-papaparse/fesm2020/ngx-papaparse.mjs:3:0-50 - Error: Module not found: Error: Can't resolve 'papaparse/papaparse.min.js' in '/path/to/project/node_modules/ngx-papaparse/fesm2020'

```
Papaparse was not added as dependency in package.json because it was not necessary with ngx-paparse@5.

Is it now necessary to add papaparse as dependency to use ngx-papaparse@6 ?

**Comment by alberthaff** [comment]
Which version of NPM are you using?

**Comment by mgoku** [comment]
I'm using yarn

Yarn v1.22.19

The problem is gone if I add papaparse as dependency in package.json

**Comment by alberthaff** [comment]
Papaparse was made a peer dependency in v6, because newer NPM versions automatically installs peer dependencies.
I'll consider moving it back into dependencies, since there are not any major gains in having it as a peerDependency vs a traditional dependency.

**Comment by VictorJSV** [comment]
Hi!.. Well I installed with npm and I had the same error
I'm using node `v14.19.0` and npm `v6.14.16`

The problem is gone if I add papaparse as dependency in `package.json` too

**Comment by gopal-augment** [comment]
I'm facing the same issue.
Node: 18.10.0
Angular: 14.2.6
ngx-papaparse: "^6.0.0"

any idea?

**Comment by gopal-augment** [comment]
Got worked with npm i papaparse as dependecy

**Comment by alberthaff** [comment]
Should be fixed as of 6.0.1
